build/ci/release.yml

```diff
@@ -226,6 +226,17 @@ functions:
           - tool_name
         working_dir: src/github.com/mongodb/mongodb-atlas-cli/build/package/docker/meta
         binary: ./docker-build.sh
+  "powershell build":
```

Comment:
could you please add a link to the patch you used to test?

Reply:
Here you go: https://spruce.mongodb.com/version/62fe47f40ae606096ab283e0/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC